api/src/test/java/org/pmiops/workbench/api/OfflineClusterControllerTest.java

```diff
@@ -81,47 +82,59 @@ public void setUp() {
     projectIdIndex = 0;
   }
 
-  private Cluster clusterWithAge(Duration age) {
+  private ListClusterResponse clusterWithAge(Duration age) {
     return clusterWithAgeAndIdle(age, Duration.ZERO);
   }
 
-  private Cluster clusterWithAgeAndIdle(Duration age, Duration idleTime) {
+  private ListClusterResponse clusterWithAgeAndIdle(Duration age, Duration idleTime) {
     // There should only be one cluster per project, so increment an index for
     // each cluster created per test.
-    return new Cluster()
+    return new ListClusterResponse()
         .clusterName("all-of-us")
         .googleProject(String.format("proj-%d", projectIdIndex++))
         .status(ClusterStatus.RUNNING)
         .createdDate(NOW.minus(age).toString())
         .dateAccessed(NOW.minus(idleTime).toString());
   }
 
-  private void stubClusters(List<Cluster> clusters) throws Exception {
+  // TODO: this is fragile.  will need a line for each field we want to test
+  private Cluster toCluster(ListClusterResponse listClusterResponse) {
+    return new Cluster()
+        .clusterName(listClusterResponse.getClusterName())
+        .googleProject(listClusterResponse.getGoogleProject())
+        .status(listClusterResponse.getStatus())
+        .createdDate(listClusterResponse.getCreatedDate())
+        .dateAccessed(listClusterResponse.getDateAccessed());
+  }
+
+  private void stubClusterListResponse(List<ListClusterResponse> clusters) throws Exception {
```

Comment:
nit: I don't think this has to change but I found it a little more intuitive to keep passing in `Cluster` objects and then used a `Cluster -> ClusterListResponse` method to modify line 111.